netskope_connector.py

```diff
@@ -1529,25 +1548,26 @@ def _handle_scim_get_groups(self, param):
             summary['total_groups'] = len(resources)
             return action_result.set_status(phantom.APP_SUCCESS)
         except Exception as e:
+            self._dump_error_log(e)
             error_msg = self._get_error_message_from_exception(e)
```

Comment:
Add error_print in starting of _get_error_message_from_exception to avoid calling _dump_error_log there

Reply:
Done